### PR TITLE
docs: Add some tutorial sections.

### PR DESCRIPTION
**Pull request by vpetrovykh**
Some basic tutorials for setting up the DB and schema, inserting and querying data, and setting up GraphQL API.

### CODE REVIEW

File: docs/tutorial/createdb.rst [hunk 1]
```diff
@@ -0,0 +1,112 @@
+.. _ref_tutorial_createdb:
+
+Create DB
+=========
+
+First step in a brand new project is to create the DB for it:
```
Comment (elprans) [inline docs/tutorial/createdb.rst]: Please use "the database" instead of "the DB".

File: docs/tutorial/createdb.rst [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+
+.. code-block:: edgeql-repl
+
+    db> CREATE DATABASE tutorial;
+    CREATE
+
+The above :ref:`command <ref_eql_ddl_databases>` creates a new
+:ref:`database <ref_datamodel_databases>` in the EdgeDB cluster. Now
```
Comment (elprans) [inline docs/tutorial/createdb.rst]: Replace "cluster" with "instance".

File: docs/tutorial/createdb.rst [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+
+
+.. _ref_tutorial_createdb_sdl:
+
+ESDL
+----
+
+The :ref:`EdgeDB schema definition language <ref_eql_sdl>` provides a
+way to describe a :ref:`migration <ref_eql_ddl_migrations>` to a
+specific state. It is great for setting up a new database because it
```
Comment (elprans) [inline docs/tutorial/createdb.rst]: _schema state_.

File: docs/tutorial/createdb.rst [hunk 4]
```diff
@@ -0,0 +1,112 @@
...
+
+
+.. _ref_tutorial_createdb_ddl:
+
+DDL
+---
+
+The :ref:`data definition language <ref_eql_ddl>` focuses on
+transforming the current schema state into the desired target step by
+step. This method is equally valid, but it is a low-level approach to
```
Comment (elprans) [inline docs/tutorial/createdb.rst]: _lower level,  and more explicit, approach_

File: docs/tutorial/createdb.rst [hunk 5]
```diff
@@ -0,0 +1,112 @@
...
+altering the schema and is less human friendly in terms of giving a
```
Comment (elprans) [inline docs/tutorial/createdb.rst]: Please drop the bit about "human friendly".  It's enough to say that DDL is a lower-level approach.

File: docs/tutorial/createdb.rst [hunk 6]
```diff
@@ -0,0 +1,112 @@
...
+clear picture of the final resulting state.
+
+Since this is an imperative language, the ``Person`` :ref:`type
```
Comment (elprans) [inline docs/tutorial/createdb.rst]: I'd write this as "In DDL the order of the definitions matters, so..."

File: docs/tutorial/createdb.rst [hunk 7]
```diff
@@ -0,0 +1,112 @@
...
+
+    tutorial> CREATE TYPE Movie {
+    .........     CREATE REQUIRED PROPERTY title -> str;
+    .........     # the year of release
+    .........     CREATE PROPERTY year -> int64;
+    .........     CREATE REQUIRED LINK director -> Person;
+    .........     CREATE MULTI LINK cast -> Person;
+    ......... };
+
+Now that the schema is setup we're ready to
```
Comment (elprans) [inline docs/tutorial/createdb.rst]: _set up_

Comment (vpetrovykh) [reply docs/tutorial/createdb.rst]: OK

File: docs/tutorial/createdb.rst (hunk 3 above)
Comment (vpetrovykh) [reply docs/tutorial/createdb.rst]: OK

File: docs/tutorial/createdb.rst (hunk 6 above)
Comment (vpetrovykh) [reply docs/tutorial/createdb.rst]: I've updated the wording.

File: docs/tutorial/createdb.rst (hunk 5 above)
Comment (vpetrovykh) [reply docs/tutorial/createdb.rst]: I've removed "human friendly". However I want to keep the bit about DDL not being transparent when it comes to figuring out the final schema state.

File: docs/tutorial/createdb.rst (hunk 4 above)
Comment (vpetrovykh) [reply docs/tutorial/createdb.rst]: OK

File: docs/tutorial/createdb.rst (hunk 2 above)
Comment (vpetrovykh) [reply docs/tutorial/createdb.rst]: OK

File: docs/tutorial/createdb.rst (hunk 1 above)
Comment (vpetrovykh) [reply docs/tutorial/createdb.rst]: OK, replaced it everywhere in the docs.

File: docs/graphql/index.rst [hunk 8]
```diff
@@ -15,6 +15,26 @@ querying. Types and Views from thee ``default`` module are exposed
...
+    .........     database := "edgedb",
+    .........     address := "127.0.0.1",
+    .........     port := 8888,
+    .........     user := "http",
+    .........     concurrency := 4,
+    ......... };
+    CONFIGURE SYSTEM
+
+This will expose GraphQL API on port 8888. Pointing your browser to
+``http://127.0.0.1:8888/explore`` will bring up a GraphiQL interface
```
Comment (1st1) [inline docs/graphql/index.rst]: Could you please make GraphiQL a link pointing to their github page?

File: docs/tutorial/createdb.rst [hunk 9]
```diff
@@ -0,0 +1,112 @@
...
+
+.. FIXME "\c" currently causes lexer errors in doc tests
+
+.. code-block::
+
+    db> \c tutorial
+    tutorial>
+
+Now we need to set up the schema. Let's set up a basic schema for a
+movies database. It will have 2 objects: movies and people who directed and
```
Comment (1st1) [inline docs/tutorial/createdb.rst]: 2 objects -> 2 types of objects

File: docs/tutorial/createdb.rst [hunk 10]
```diff
@@ -0,0 +1,112 @@
...
+transaction is committed the schema is updated and we're ready to
+:ref:`populate the database with data <ref_tutorial_queries>`.
+
+
+.. _ref_tutorial_createdb_ddl:
+
+DDL
+---
+
+The :ref:`data definition language <ref_eql_ddl>` focuses on
```
Comment (1st1) [inline docs/tutorial/createdb.rst]: Please add a separate paragraph saying that the entire DDL section is an alternative approach to SDL and need not be followed if the reader has completed the steps in the previous section. Make it in bold font.

File: docs/tutorial/graphql.rst [hunk 11]
```diff
@@ -0,0 +1,185 @@
...
+          },
+          {
+            "title": "Dune",
+            "year": null
+          }
+        ]
+      }
+    }
+
+It's possible apply a :ref:`filter <ref_graphql_overview_filter>` to
```
Comment (1st1) [inline docs/tutorial/graphql.rst]: "It's possible apply" -> "It's possible **to** apply" ?

File: docs/tutorial/queries.rst [hunk 12]
```diff
@@ -0,0 +1,274 @@
...
+    .........             first_name := 'Ana',
+    .........             last_name := 'de Armas',
+    .........         }),
+    .........     }
+    ......... };
+    {Object { id: <uuid>'4d0c8ddc-54d4-11e9-8c54-7776f6130e05' }}
+
+.. note::
+
+    The specific ``id`` values will be different. They are show
```
Comment (1st1) [inline docs/tutorial/queries.rst]: show -> shown

File: docs/graphql/index.rst (hunk 8 above)
Comment (vpetrovykh) [reply docs/graphql/index.rst]: OK

File: docs/tutorial/createdb.rst (hunk 9 above)
Comment (vpetrovykh) [reply docs/tutorial/createdb.rst]: OK

File: docs/tutorial/graphql.rst (hunk 11 above)
Comment (vpetrovykh) [reply docs/tutorial/graphql.rst]: OK

File: docs/tutorial/queries.rst (hunk 12 above)
Comment (vpetrovykh) [reply docs/tutorial/queries.rst]: OK

File: docs/tutorial/createdb.rst (hunk 10 above)
Comment (vpetrovykh) [reply docs/tutorial/createdb.rst]: OK

**Pull request merged**